hosted update on homepage and mark page (#3)
geostyler/geostyler-cookbook: index.md

---
layout: home
---
# 🖥️ Demo

> All written functions can be tested here Geostyler [Official demo](https://geostyler.github.io/geostyler-demo/) UI. Also checkout [repo](https://github.com/geostyler/geostyler-demo) for the same.

Reviewer: ```suggestion
All written functions can be tested on the [GeoStyler Demo](https://geostyler.github.io/geostyler-demo/). Also checkout the [geostyler-demo repo](https://github.com/geostyler/geostyler-demo) for the same.
```

> # 🎨 Getting familiar with UI

Reviewer: ```suggestion
# 🎨 Getting familiar with the UI
```

> [Geostyler](https://geostyler.org/) [Official demo](https://geostyler.github.io/geostyler-demo/) is a very easy to use, interactive web app which allows user to start testing capabilities of the application without installing it. 

Reviewer: ```suggestion
[Geostyler Demo](https://geostyler.github.io/geostyler-demo/) is a very easy to use, interactive web app which allows users to start testing capabilities of the [GeoStyler library](https://github.com/geostyler/geostyler) without installing it.
```

There are various components to it as follows 

![demo]( images/demo.png "demo")

## 1. Language

> [Geostyler](https://geostyler.org/) demo UI can be translated in 4 different languages 

Reviewer: Soon to be 5 :smile:. See https://github.com/geostyler/geostyler/pull/1731

Author: done I have added

- EN - English
- DE - German
- ES - Spanish

> - 中文 - mandarin

Reviewer: ```suggestion
- 中文 - Mandarin
```

## 2. Compact

Toggling this changes the UI for `Graphical Editor` 7️⃣ into 2 different modes. Please note that from functionalities stand point it will not make any difference. 

Here are both modes

### Compact mode

![compact]( images/compact.png "compact")

### Expanded mode

![expand]( images/expanded.png "expand")

## 3. Symbolizer Renderer

> Currently [Geostyler](https://geostyler.org/) supports `Openlayers` and `SLD` rending for the symbol. Changing this setting will reflect the way symbol is rendered in the `Graphical Editor`

Reviewer: ```suggestion
Currently, [Geostyler](https://geostyler.org/) supports `Openlayers` and `SLD` rendering of symbols. Changing this setting will reflect the way symbols are rendered in the `Graphical Editor`
```

![renderer]( images/renderer.png "renderer")

## 4. Load Style

This allows users to import existing `SLD` created using Geoserver, QGIS, etc in the application. 

> Once loaded users can make changes if required or visualize it as it is. 

Reviewer: ```suggestion
Once loaded, users can make changes if required or visualize it as it is. 
```
